docs/content/preview/yugabyte-voyager/reference/assess-migration-bulk.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Bulk assess migration
+headcontent: yb-voyager assess-migration-bulk
+linkTitle: Bulk assess migration
+description: YugabyteDB Voyager bulk assess migration reference
+menu:
+  preview_yugabyte-voyager:
+    identifier: assess-migration-bulk-voyager
+    parent: assess-migration-voyager
+    weight: 1
+badges: tp
+type: docs
+---
+
+[Assess the migration](../../migrate/assess-migration) from multiple source (Oracle) database schemas to YugabyteDB.
+
+## Syntax
+
+```text
+Usage: yb-voyager assess-migration-bulk [ <arguments> ... ]
+```
+
+### Arguments
+
+The valid *arguments* for assess migration are described in the following table:
+
+| Argument | Description/valid options |
+| :------- | :------------------------ |
+| &#8209;&#8209;bulk&#8209;assessment&#8209;dir | Directory path where assessment data is output. |
+| --continue-on-error | Print the error message to the console and continue to next schema assessment.<br>Default: true.<br>Accepted parameters: true, false, yes, no, 0, 1. |
+| --fleet-config-file | Path to the CSV file with connection parameters for schema(s) to be assessed. The first line must be a header row. <br>Fields (case-insensitive): 'source-db-type', 'source-db-host', 'source-db-port', 'source-db-name', 'oracle-db-sid', 'oracle-tns-alias', 'source-db-user', 'source-db-password', 'source-db-schema'.<br>Mandatory: 'source-db-type', 'source-db-user', 'source-db-schema', and one of ['source-db-name', 'oracle-db-sid', 'oracle-tns-alias']. |
```

Comment:
we can add the rest of part also, or maybe skip sample example:
```
 Guidelines:
                                     	- The first line must be a header row.
                                     	- Ensure mandatory fields are included and correctly spelled.

                                     Sample fleet_config_file:
                                     	source-db-type,source-db-host,source-db-port,source-db-name,oracle-db-sid,oracle-tns-alias,source-db-user,source-db-password,source-db-schema
                                     	oracle,localhost,1521,orclpdb,,,user1,password1,schema1
                                     	oracle,localhost,1521,,orclsid,,user2,password2,schema2

```

Reply:
I put the example below, it doesn't format very nicely in the table.
The header line is included.
Including mandatory fields and spelling things correctly seemed redundant.